src/main/scala/wandou/avpath/Evaluator.scala

```diff
@@ -479,7 +509,8 @@ object Evaluator {
               var i = 0
               while (values.hasNext) {
                 val value = values.next
-                elems(i) = Ctx(value, elemType, topLevelField, Some(TargetArray(arr, i, schema)))
+                // TODO not sure
```

Comment:
We are on a list, we name each sub element like the parent.
It should not be troublesome, but it's not right.

Reply:
:point_up: say it in the comment!